pyanaconda/dbus_addons/baz/baz.py

```diff
@@ -33,3 +36,29 @@ def publish(self):
         """Publish the module."""
         DBus.publish_object(BAZ.object_path, BazInterface(self))
         DBus.register_service(BAZ.service_name)
+
+    def install(self):
+        """Return installation tasks."""
+        return [self.publish_task(BAZ.namespace, BazTask())]
```

Comment:
This is not a good name of the method. It will only return list of tasks but the name says it is going to start the installation.

Please rename this method.

Reply:
I think methods requiring tasks should be named to indicate it for clients - I am fine with WithTask suffix.

I don't think changing the DBus API by adding the suffix for methods that started to require task would be less hassle for clients than eg requiring them to check if the method (which methods need to be checked? all?) returns task path (or empty path in case the method does not require tasks). Or maybe the clients should check if the task path is empty in any case (ie the opposite change, method not requiring task any more). So should we stay with names without suffix and having the methods returning task (real or empty) always?

Maybe let's start with the suffix and when we have more of the API, before publishing the API, decide if another solution makes sense?

Reply:
What about Blivet style `DoIt`, just joking :D

What about simple `run`?

Reply:
I agree with @rvykydal, let's start with the suffix.

Reply:
I agree with the suffix as well, definitely enough for the time being.